### DISCUSSION TITLE
Error while starting up TerriaMap:  ﻿Failed to load resource: net::ERR_INCOMPLETE_CHUNKED_ENCODING, may have to do with MacOS upgrade

### DISCUSSION THREAD

**Discussion by angelikaSajani** [TerriaJS/terriajs#7052] [Q&A] [answered]
As the title states, I am getting this error while my TerriaMap adaptation starts up. It seems to only happen when testing locally, and I have recently updated to MacOS 14 (Sonoma), which may be the reason. It does not seem to happen with the deployed instance, which is running in docker container. While I am glad that at least it still works remotely, this is debilitating as the turnaround time for tests on our preproduction site is prohibitively long.
It is not 100% reproducible, in the sense that sometimes a complete restart of my machine combined with a rebuild of TerriaMap fixes the problem at least temporarily (but never for long).
Has anybody experienced such problems? Any workarounds or advice?
<img width="959" alt="Screenshot 2024-02-17 at 12 06 40 pm" src="https://github.com/TerriaJS/terriajs/assets/71240254/f2f92dde-104c-4a24-bb7c-14c772824ec4">

**Comment by angelikaSajani** [comment, marked as answer]
I think this was a local issue. In the end I had to restart my remote ssh server, that seemed t fix it.

**Reply by nf-s** [reply]
Great! Thanks for posting a fix to your issue. I will check with the team to see if anyone else has had this issue

**Reply by angelikaSajani** [reply]
A bit more information: In the end I had to stop using a remote (ssh) server for development. The server is implemented as a Virtual Machine under UnRaid, is running Ubuntu 20, and in the console I consistently see the following warning: 

`2024-02-21T01:38:29.180637Z qemu-system-x86_64: warning: 9p: degraded performance: a reasonable high msize should be chosen on client/guest side (chosen msize is <= 8192). See https://wiki.qemu.org/Documentation/9psetup#msize for details.` 

This may or may not have anything to do with it, but it looks likely. The server works fine for all my other projects, but since moving my TerriaMap development to my local machine (MacOS 14) these problems have completely disappeared. 

Unfortunately I cannot see any options in the VMS setup to prevent that warning.
